src/main/java/com/technocredits/orangeHRM/pages/LoginPage.java

```diff
@@ -1,21 +1,33 @@
 package com.technocredits.orangeHRM.pages;
 
-public class LoginPage {
-
-	private static LoginPage loginPage;
-
-	private LoginPage() {
-
+import java.io.IOException;
+import java.util.Properties;
+
+import com.technocredits.orangeHRM.base.PredefinedMethods;
+import com.technocredits.orangeHRM.util.PropertyFileReader;
+
+public class LoginPage extends PredefinedMethods{
+	private static LoginPage loginPage = null;
+	Properties loginPageProperties;
+	
+	private LoginPage() throws IOException{
+		initializeBrowser();
+		PropertyFileReader propreader = new PropertyFileReader();
+		loginPageProperties = propreader.initializePropertyFile("LoginPageProperties");
 	}
-
-	public static LoginPage getInstance() {
-		if (loginPage == null)
+	
+	public static LoginPage getInstance() throws IOException{
```

Comment:
Add comment above every method.

Reply:
To create instance for LoginPage, using singleton design pattern, if Object is already created, return that, else create new instance.